screens/product-list/product-list.html

```diff
@@ -0,0 +1,381 @@
+                <button class="button-icon rounded-circle center-content-1 m-sm-r">
+                    <span class="material-icons material-icons--notification">
+                        shopping_cart
+                    </span>
+                    <span class="icon-badge rounded-circle center-content bold-font">1</span>
+                </button>
+            </nav>
+        </div>
+        <img src="/assets/images/Ayushman.jpg" class="avatar avatar--sm rounded-circle m-sm-r" alt="Avatar" />
+    </header>
+    <!-- Header navigation end -->
+    <div class="menu">
+        <button class="menu__button center-content btn btn--primary-text p-xs-vr p-sm-hr rounded-circle semibold-font">
+            <i class="material-icons" aria-hidden="true">
+                menu
+            </i>
+        </button>
+    </div>
+    <!-- Siderbar navigation start -->
+    <div class="content">
+        <aside class="drawer drawer--permanent">
+            <nav>
+                <section class="filter m-xs-all">
+                    <h4>Filters</h4>
+                    <button class="btn btn--default-text p-xs-vr p-sm-hr rounded-sm bold-font">CLEAR</button>
+                </section>
+
+                <section class="category m-xs-all text-grey-500">
+                    <div>
+                        <h4>Category</h4>
+                    </div>
+                    <ul class="list">
+                        <li class="list__input m-xs-all">
+                            <label for="id-3" class="center-content">
+                                <input id="id-3" type="checkbox" class="m-xs-r" checked>
+                                Indian Jwellary
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Earrings
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Gold
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Diamond
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Rings
+                            </label>
```

Comment:
- Here you have given same id for multiple labels.
- So whenever i am trying to select gold , diamond or rings it is selecting only second label i.e Earrings.
- You can correct this by giving different id's